execution/execution.go

```diff
@@ -1,10 +1,11 @@
 package execution
 
 import (
+	"encoding/hex"
 	"time"
 
 	"github.com/mesg-foundation/core/service"
-	"github.com/mesg-foundation/core/x/xstructhash"
+	"github.com/mesg-foundation/core/utils/structhash"
```

Comment:
You are using this new package introduced by https://github.com/mesg-foundation/core/pull/730 but not merged yet.